pallets/subtensor/src/root.rs

```diff
@@ -577,6 +577,135 @@ impl<T: Config> Pallet<T> {
         Ok(())
     }
 
+    pub fn do_set_root_weights(
+        origin: T::RuntimeOrigin,
+        netuid: u16,
+        hotkey: T::AccountId,
+        uids: Vec<u16>,
+        values: Vec<u16>,
+        version_key: u64,
+    ) -> dispatch::DispatchResult {
+        // --- 1. Check the caller's signature. This is the coldkey of a registered account.
+        let coldkey = ensure_signed(origin)?;
+        log::info!(
```

Comment:
```suggestion
        log::trace!(
```

Reply:
All similar logs use `log::info!` including in the regular `do_set_weights`.